templates/ez_segmenter.ipynb

```diff
@@ -0,0 +1,1019 @@
+{
```

Comment:
<p>Instead of providing instructions in code comments, use a markdown cell above to document.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/958/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>Done</p>